pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -672,7 +676,7 @@ func (r *ReconcileInfinispan) statefulSetForInfinispan(m *infinispanv1.Infinispa
 			Name:        m.Name,
 			Namespace:   m.Namespace,
 			Annotations: consts.DeploymentAnnotations,
```

Comment:
@rigazilla could you please review the deployment annotation marked in #488
This is good issue to change it too

Reply:
Removed all the annotation without prefix
Removed version